src/openrct2/drawing/Drawing.Sprite.cpp

```diff
@@ -634,7 +634,7 @@ void FASTCALL gfx_draw_sprite_raw_masked_software(
     }
 
     // Only BMP format is supported for masking
```

Comment:
Update comment to state that must have transparency. Tbh there is a bug here as it also must not be rle but we will leave that for another day.